backend/src/main/java/de/qaware/mercury/business/image/color/analysis/impl/BorderImageAnalyzerImpl.java

```diff
@@ -0,0 +1,66 @@
+package de.qaware.mercury.business.image.color.analysis.impl;
+
+import de.qaware.mercury.business.image.color.Color;
+import de.qaware.mercury.business.image.color.ColorMap;
+import de.qaware.mercury.business.image.color.analysis.ColorHistogramCreator;
+import de.qaware.mercury.business.image.color.analysis.graphicsmagick.GMImageToTextConverter;
+import de.qaware.mercury.business.image.color.analysis.graphicsmagick.GMTextOutputParser;
+import org.springframework.stereotype.Service;
+
+import java.io.InputStream;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * This ImageAnalyzer implementation creates a histogram of the image by only looking at
+ * the border of the image. The border is most relevant if you wish to embed the image in
+ * a matching background color.
+ */
+@Service
+public class BorderImageAnalyzerImpl implements ColorHistogramCreator {
+
+    private final GMImageToTextConverter gmImageToTextConverter;
```

Comment:
Willst du die in den Tests mal mocken oder wieso sind da Interfaces dazwischen? Ich würd's verstehen, wenn das Interface ImageToTextConverter heißt, aber das hat ja sogar die Implementierung mit im Namen.

Reply:
Das war tatsächlich dafür, dass man das in nem Test mocken kann. Jetzt gibts den Test auch ;)